TeamCreator/Modules/Dashboard/View/Cell/DashboardCollectionViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SecondCollectionViewCell.swift
+//  TeamCreator
+//
+//  Created by Giray Aksu on 31.07.2024.
+//
+
+import UIKit
+
+class DashboardCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var backgroundImageView: UIImageView!
+    @IBOutlet weak var categoryLabel: UILabel!
+
+    static let identifier = "SecondCollectionViewCell"
```

Comment:
Hard coded stringlerimiz için bir Constant dosyamız veya Contant enumımız olsaydı ve orda toplasaydık daha güzel olabilir miydi ?